vnext/ReactWindowsCore/ReactWindowsCore.vcxproj

```diff
@@ -98,6 +98,14 @@
       </AdditionalIncludeDirectories>
       <AdditionalIncludeDirectories Condition="'$(USE_HERMES)'=='true'">$(HERMES_Package)\installed\$(VcpkgTriplet)\include\;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessToFile>false</PreprocessToFile>
+      <AdditionalOptions Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">/await /permissive- %(AdditionalOptions)</AdditionalOptions>
```

Comment:
>/permissive- [](start = 88, length = 12)

/permissive- is already part of Warnings.props

Reply:
Yeah it's crazy how bad VS is at editing project files :(